src/stub.rs

```diff
@@ -33,9 +33,9 @@ pub enum ReadStub {
 }
 
 #[derive(Debug)]
-enum Stub {
-  Read(Vec<ReadStub>),
-  Loop(Box<Stub>),
-  LoopLine(Box<Stub>),
+pub enum Stub {
+  Read(Vec<VariableStub>),
+  Loop { count: String, command: Box<Stub> },
+  LoopLine { object: String, variables: Vec<VariableStub> },
   Write(String),
 }
```

Comment:
You will need to add a Statement to that enum if you want to close #13 

This is clash that uses it: https://www.codingame.com/contribute/view/653884ada8d44d632a5f42168453bf565049a